.readthedocs.yml

```diff
@@ -0,0 +1,10 @@
+formats: ['htmlzip']
```

Comment:
Any reason for this? HTML and JSON will always be built by default so I use:

```yaml
# Don't build any extra formats, HTML and JSON will
# always be built by default.
formats:
  - none

requirements_file: docs/requirements.txt

build:
  image: latest

python:
  pip_install: true
  setup_py_install: false
  version: 3.6
```

whenever I have a RTD config.

Reply:
I include this format because that is how the docs are built right now, but we can drop it

Reply:
Let's do that.

Reply:
Done!